### DISCUSSION TITLE
Apache Superset  1.5.0 login error

### DISCUSSION THREAD

**Discussion by JohnRson** [apache/superset#19801] [Q&A / Help]
I have been using Apache Superset 1.4.2 and wanted to upgrade to 1.5.0 as it supports non-timeseries-x-axis.

https://github.com/apache/superset/tree/master/RELEASING/release-notes-1-5

I tried to install it from github via python on windows 10 into an unique environment:

conda activate %env%
call conda install git

pip install --upgrade setuptools pip  --user python
pip install X:\python_geohash-0.8.5-cp39-cp39-win_amd64.whl
pip install markupsafe==2.0.1
pip install sqlalchemy-trino

pip install git+https://github.com/apache/superset.git@1.5.0rc3
pip show apache-superset

SET FLASK_APP=superset
superset db upgrade
flask fab create-admin
superset load_examples
superset init
superset run -p 8088 --with-threads --reload --debugger

I already installed some aditional libraries (geohash, markupsafe, sqlalchemy-trino), otherwise it wouldn't finish.
So far, there is no console error but when I open http://127.0.0.1:8088/superset/welcome/ it just shows an empty white screen 

![Bild1](https://user-images.githubusercontent.com/104075422/164315281-b8e1b388-ad87-4b1c-95f0-7c7caba35d47.png)

Does anyone know how to fix this issue?

**Comment by villebro** [comment]
Hi @JohnRson, unfortunately Superset does not officially support Windows, so we cannot guarantee it works on Windows 10. Therefore I'm going to convert this to a discussion. But we're happy to accept PRs to improve support for unsupported platforms.
